.ci.yaml

```diff
@@ -699,7 +699,6 @@ targets:
 
   - name: Linux plugin_test_linux
```

Comment:
Led run succeeds with the new cmake version: https://ci.chromium.org/raw/build/logs.chromium.org/flutter/led/keyonghan_google.com/d813b8f1ffe0863aff0697eaff6a6e86b51e2a8af06502fcb345af4e7e523afb/+/build.proto?server=chromium-swarm.appspot.com

Updating the cmake version to `version:2@3.25.2.chromium.6` should pass the CI.

Reply:
Great! That has to be a separate PR that lands and then propagates I assume, like other CI in yaml changes?

Reply:
Presubmit CI will pick up the changes right away, so changes in this same PR will work.

Please allow https://flutter-review.googlesource.com/c/recipes/+/40060 to land before pushing the change, as the infra config complains about the special character `@` in the version: https://ci.chromium.org/ui/p/flutter/builders/try/Linux%20ci_yaml%20flutter%20roller/2119/overview.

After the above CL lands, please use version `build_id:8787856497187628321` instead of `version:2@3.25.2.chromium.6`.
(successful led run with version `build_id:8787856497187628321`: :https://ci.chromium.org/raw/build/logs.chromium.org/flutter/led/keyonghan_google.com/9a8f995890c8aa1dd97b86de8a5f3a2018dcce8050fcc2f2143d6b6569aa0e5c/+/build.proto?server=chromium-swarm.appspot.com)

Reply:
> Presubmit CI will pick up the changes right away, so changes in this same PR will work.

Ah, nice. Is there a list somewhere of what does and doesn't work in-PR? I know that timeout changes don't take effect right away, for instance, having been bitten by that before, but I'm not clear on what things do.

Reply:
All properties should be picked up right away. Updated https://github.com/flutter/cocoon/pull/2519 to make it clear.